File: Moderate/swapper.go
package main
import (
  "fmt"
)

func swap(x, y int) (int, int) {
  return y, x
}

func main() {
  var x, y int = 3, 5
  fmt.Println(x, y)

  x, y = swap(x, y)
  fmt.Println(x, y)
}

File: Moderate/swapper.js
// swap a number in place without temporary variable

function swapper(numOne, numTwo){
  numOne = numOne - numTwo;
  numTwo = numTwo + numOne;
  numOne = numTwo - numOne;

  return [numOne, numTwo];
}

console.log(swapper(1, 3))
console.log(swapper(5, 3))
